addon/templates/components/nypr-m-byline.hbs

```diff
@@ -3,7 +3,7 @@
   By
   {{#each @authors as |author|}}
     {{~#if (and (not (has-next author @authors)) (gt @authors.length 1))~}}
-     &nbsp;and&nbsp;
+      and&nbsp;
```

Comment:
Not sure if this was deleted on purpose but the &nbsp; is here to make the spacing in the template output very explicit, because the `~` characters in the template statement cause the whitespace to collapse.

Not actually sure if we really need to collapse all the whitespace in this case (it's more useful for things like inline lists, where you don't want a space character between list items) but that's what it's doing.

https://handlebarsjs.com/guide/expressions.html#whitespace-control